content/zh/docs/concepts/observability-primer.md

```diff
@@ -61,7 +61,7 @@ I, [2021-02-23T13:26:23.505892 #22473]  INFO -- : [6459ffe1-ea53-4044-aaa3-bf902
 
 ### Span 属性
```

Comment:
Add a heading ID here. For details, see https://opentelemetry.io/docs/contributing/localization/#headings.